packages/framer-motion/src/animation/GroupPlaybackControls.ts

```diff
@@ -0,0 +1,29 @@
+import { AnimationPlaybackControls } from "./types"
+
+export class GroupPlaybackControls implements AnimationPlaybackControls {
+    animations: AnimationPlaybackControls[]
+
+    constructor(animations: Array<AnimationPlaybackControls | undefined>) {
+        this.animations = animations.filter(
+            Boolean
+        ) as AnimationPlaybackControls[]
+    }
+
+    get currentTime() {
+        return this.animations[0].currentTime
+    }
```

Comment:
Shouldn't it be the `max()` of currentTimes or something?

Reply:
That would be correct for `duration`, as I realised after implementing `max` originally too. In theory all these animations should be in sync, some could be interrupted at which point this is a bit of a fudge, we probably want to filter out animations that aren't running or paused and take the duration from the first one of those. But I do want to take a second look at `currentTime` before making an official API, this is just for internal use and it doesn't currently represent more than one animation, but I'll add a comment